### ISSUE TITLE
Safari could not open the page because the server stopped responding.

### ISSUE THREAD

**Issue by GalaxiousCosmo** [wileyyugioh/Osu-Touchpad#2]
Hi, not sure if I'm necro-ing a dead project, but I thought it'd be worth a try anyways. I downloaded the code and extracted it, ran "nw.exe", added the firewall exception for "nw.exe" through private and public networks, scanned the QR code in the iOS camera app and opened it in Safari.

The page loads for a while, then eventually Safari returns "Safari could not open the page because the server stopped responding." I'm not sure what I'm doing wrong.

I'm running Win10, iPad Pro (2018), iOS 13.3.1
Thanks in advance!

**Comment by wileyyugioh** [comment]
I just tested it, and the app worked as expected. Can you double check that:

1. The iPad and the computer are connected to the same wifi network
2. You're able to connect via localhost and ip address to the app on the pc

**Comment by GalaxiousCosmo** [comment]
Hey, thanks for your reply. I just tested it again after seeing what you said, for some reason it's working now, despite it persistently not working 2 weeks ago. I haven't made any changes at all. However, I'm encountering an incredible amount of input lag consistently, making it unusable. I tried using it under various conditions, but even with every other application on my pc closed, as well as every other device disconnected from the wifi network, the input lag is still in the region of 0.5 - 1 second at minimum. I see that your replies on the [old reddit thread](https://www.reddit.com/r/osugame/comments/41fwci/use_your_mobile_device_as_an_osu_tablet/) say that 

> the main limiting factor is the speed between the router and your devices

so I suppose there's simply a huge amount of variance depending on the user conditions, I guess I'll have to give it up then.

Also, I noticed a few other issues which may be of interest to you. Moving to the edges of the iPad barely moves the cursor to the edge of the screen at all. This is fixed by using the startAutoYPos as described in the readme, but as soon as another application is switched to, Osu for example, it reverts back to the old behaviour instantly, at least in my case.

With regards to input lag, I found it was significantly worse using the Apple Pencil 2 compared to simply using touch input, somewhere in the region of 2x more input delay. In any case, the input lag is simply too high. In the reddit thread, I see that you said this isn't really a complete tablet substitution, in any case I'm now considering buying a tablet despite using a mouse for years (circular mouse movement causing sensor drift is a plague).
